content/information-distribution/node.md

```diff
@@ -54,14 +57,65 @@ the Matrix room to join.
 Once a Matrix room is joined, client implementations **must** retrieve and save
 the [list of
 aliases](https://matrix.org/docs/spec/client_server/r0.4.0.html#m-room-aliases)
-for this room. In the event of an entry node that's currently being used being
-unreachable, clients implementations and **must** use this list to provide users
-with an updated list of alternative entry nodes and aliases to use. The address
-of the node managing a given alias can be retrieved through a [DNS
-query](https://github.com/matrix-org/synapse/#setting-up-federation). Client
-implementations **can** check for pings in order to filter out the inactive
-nodes from this list.
+for this Matrix room. This can be done by processing all of the `m.room.aliases`
+events in the state of the Matrix room, each of them containing the list of
+aliases defined for a given node. The content for such an event looks like this:
+
+```
+"aliases": [
+    "#alias1:example.com",
+    "#alias2:example.com"
```

Comment:
I think the example would need aliases from other nodes

Reply:
One `m.room.aliases` events only contains the aliases for one given Matrix homeserver, so I can't really do that. Might be unclear, though, if you misunderstood it.

Reply:
Fixed in https://github.com/Informo/specs/pull/2/commits/4bb407f4ca3b9f2a14c1fbc01cc977427eca4810.